resources/themes/cura-light/theme.json

```diff
@@ -433,6 +441,24 @@
         "objects_menu_size": [20, 40],
         "objects_menu_size_collapsed": [20, 17],
         "build_plate_selection_size": [15, 5],
-        "objects_menu_button": [0.3, 2.7]
+        "objects_menu_button": [0.3, 2.7],
+
+        "toolbox_thumbnail_small": [6.0, 6.0],
+        "toolbox_thumbnail_medium": [8.0, 8.0],
+        "toolbox_thumbnail_large": [12.0, 12.0],
+        "toolbox_footer": [1.0, 4.5],
+        "toolbox_footer_button": [8.0, 2.5],
+        "toolbox_showcase_spacing": [1.0, 1.0],
+        "toolbox_header_tab": [8.0, 4.0],
```

Comment:
I'm thinking that the toolbox header theming should be merged with the top bar theming. They should look/feel the same, so we can enforce that.

Reply:
You're killing me here Ruben! After all that conversation I added this so that this part of the UI is individually theme able if the user wants to theme it differently.